### PR TITLE
Permissions migration script

### PR DESCRIPTION
**Pull request by sofiarm21**
### **Issue #379**

**Important:**

This branch was branched off from #382 so have all the changes of that branch

**Description:**

This pr consists of making and running a script that will create permission for the relation contributor-project. These permissions will come from GitHub as described in #382 

**Breakdown**:

* I recommend reading #382 pr description. 
* Besides using all logic implemented in #382 to run this script you need to go to the apollo console and run `syncContributorsPermissions` that takes  the GitHub API key from the contributor as parameter making the request and will return the number of contributors
* This will sync the permission relation between contributor-project but not contributor-client

**Implementation proof:**

https://www.loom.com/share/c8e90a71cf1a4bdf98e6143adf68ff49

### CODE REVIEW

File: api/schema/resolvers/PermissionResolver.js [hunk 1]
```diff
@@ -32,6 +34,25 @@ module.exports = {
...
+    },
+    Mutation: {
+        syncContributorsPermissions: async (root, { github_access_token }, { models }) => {
+            const contributors = await models.Contributor.findAll({
+                raw: true
+            })
+            await Promise.all(contributors.map(async c => {
+                if (c.github_access_token) {
+                    const githubContributor = {
+                        accessToken: github_access_token
```
Comment (otech47) [inline api/schema/resolvers/PermissionResolver.js:47]: Do we need to pass an access token manually?

**Review by otech47 — CHANGES_REQUESTED** [review]
Let's check on this again to see if we can reduce the dependency on the manual `github_access_token`

**Review by otech47 — CHANGES_REQUESTED** [review]
Use the github token of the owner of the Trinary Project to get permission levels and try to populate the Permissions table from that

**Comment by sofiarm21** [comment]
Too many requests are being made
contributors (106) * projects (13) + 1 project owner
Screen Shot 2021-06-17 at 11 54 06 PM

**Review by otech47 — APPROVED** [review]

**Pull request merged**